env0/test_helpers.go

```diff
@@ -0,0 +1,47 @@
+package env0
+
+import (
+	"fmt"
+)
+
+// TFSource Enum
+type TFSource string
+
+const (
+	DataResource TFSource = "data"
+	Resource     TFSource = "resource"
+)
+
+func DataSourceAccessor(resourceType string, resourceName string) string {
```

Comment:
Please make sure all functions in this file are private

Reply:
fix string newlines + CR 41b1ae8